pkg/providers/instancetype/suite_test.go

```diff
@@ -813,6 +813,58 @@ var _ = Describe("InstanceType Provider", func() {
 			}
 		})
 
+		It("should support individual instance type labels", func() {
+			ExpectApplied(ctx, env.Client, nodePool, nodeClass)
+
+			nodeSelector := map[string]string{
+				// Well known
+				v1.LabelTopologyRegion:           fake.Region,
+				corev1beta1.NodePoolLabelKey:     nodePool.Name,
+				v1.LabelTopologyZone:             fmt.Sprintf("%s-1", fake.Region),
+				v1.LabelInstanceTypeStable:       "Standard_NC24ads_A100_v4",
+				v1.LabelOSStable:                 "linux",
+				v1.LabelArchStable:               "amd64",
+				corev1beta1.CapacityTypeLabelKey: "on-demand",
+				// Well Known to AKS
+				v1alpha2.LabelSKUName:                      "Standard_NC24ads_A100_v4",
+				v1alpha2.LabelSKUFamily:                    "N",
+				v1alpha2.LabelSKUVersion:                   "4",
+				v1alpha2.LabelSKUStorageCacheSize:          "0", // Either Temp disk or cache disk can be empty :)
+				v1alpha2.LabelSKUStorageTempMaxSize:        "0",
+				v1alpha2.LabelSKUStorageEphemeralOSMaxSize: "53.6870912",
+				v1alpha2.LabelSKUAcceleratedNetworking:     "true",
+				v1alpha2.LabelSKUEncryptionAtHostSupported: "true",
+				v1alpha2.LabelSKUStoragePremiumCapable:     "true",
+				v1alpha2.AKSLabelCluster:                   "test-cluster",
```

Comment:
This one is a bit special; we need a better way to handle it. For now maybe just put it last (or in a separate category?) so it does not get lost ...